docs/docs/dev/security/certificates.md

```diff
@@ -122,6 +122,46 @@ You can also provide the CA via environment variable `KUMA_CONTROL_PLANE_CA_CERT
 
 See [Data plane proxy authentication](./dp-auth.md) and [Zone Ingress authentication](./zone-ingress-auth.md).
 
+## Prometheus to control plane communication
+
+It is possible to enabled TLS on the Monitoring Assignment Discovery Service.
```

Comment:
as a first-time reader unfamiliar with the product, I don't know what it is. A quick search for "Monitoring Assignment Discovery Service" does not help :(